pkg/controller/disruption/disruption.go

```diff
@@ -446,20 +460,20 @@ func (dc *DisruptionController) Run(ctx context.Context) {
 	<-ctx.Done()
 }
 
-func (dc *DisruptionController) addDb(obj interface{}) {
+func (dc *DisruptionController) addDb(logger klog.Logger, obj interface{}) {
 	pdb := obj.(*policy.PodDisruptionBudget)
-	klog.V(4).Infof("add DB %q", pdb.Name)
-	dc.enqueuePdb(pdb)
+	logger.V(4).Info("Add DB", "PodDisruptionBudget", klog.KObj(pdb))
+	dc.enqueuePdb(logger, pdb)	
```

Comment:
Watch out for unnecessary spaces.

Run `hack/update-gofmt.sh` when you editor doesn't handle this.

But ultimately you should better configure your editor to work well with Go, which typically includes live syntax checking and running "gofmt" on saving the file.